### DISCUSSION TITLE
А давайте сделаем мегафон юзабельным...

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#818] [Предложения]
Речь пойдет о полицейском мегафоне, установленным в машине. А именно про его использование через войс чат.

Предлагаю увеличить его радиус действия, ибо сейчас, как мне кажется, он очень мал.
И сделать звуковой эффект по типу этого - https://vk.com/video-183094000_456239317

Это будет реально круче, чем бинды с заготовленным текстом

**Comment by Wani4ka** [comment]
Полторы тысячи юнитов - это прям ОЧЕНЬ много, по идее.
А звуковой эффект.. Поясни, пожалуйста, как ты хочешь, чтобы эта механика с заготовленными фразами сочеталась с возможностью писать текстом и говорить в войс-чат через громкоговоритель

**Reply by KillerBunny031** [reply]
Я, наверное, неправильно выразился по поводу звукого эффекта. 

Сейчас, когда говорят через мегафон, этого непонятно. Может офицер просто кричит, а может говорит через него. Проверить это никак нельзя. 

Это видео я вставил только как пример самого звука. Я не нашел другого видео для примера(
Т. е., чтобы, когда офицеры говорили в мегафон был слышен этот эффект, что они говорят через него, а сейчас это просто увеличенный радиус разговора. 

Я видел, как некоторые офицеры на ДБГ самостоятельно пытались делать что-то подобное. 
Вот тут плохо слышно, но все же - https://youtu.be/QWddWsmF3nk
На моменте 20:05 слышен "приятель" с эффектом мегафона. 

Еще было бы неплохо сделать мегафон на отдельную кнопку, ведь сейчас он стоит на shift, а это стандартная кнопка для ускорения авто, что не совсем удобно.
